### ISSUE TITLE
Add comment not working

### ISSUE THREAD

**Issue by JamesInform** [requarks/wiki#5222]
Using: Wiki.JS 2.5.278 /  (Docker Image)

When trying to add a comment to a page nothing happens.

The browser internally throws the following javascript exception:
![image](https://user-images.githubusercontent.com/38714600/165274207-c84d21b3-ceac-41d0-9ea0-2d0582e86578.png)

**Comment by Asgoret** [comment]
Same on 2.5.278. On 2.5.277 all works fine

**Comment by ponceto** [comment]
Hi,

Same problem here.

The JavaScript Console shows this error message :

```
TypeError: document.querySelector(...) is null
    goToComments https://wiki.xxxxxxxxxxxxxxxx.com/_assets/js/theme0.js?1650156247:2
    click https://wiki.xxxxxxxxxxxxxxxx.com/_assets/js/theme0.js?1650156247:2
    Wt https://wiki.xxxxxxxxxxxxxxxx.com/_assets/js/app.js?1650156247:478
    n https://wiki.xxxxxxxxxxxxxxxx.com/_assets/js/app.js?1650156247:478
    Wt https://wiki.xxxxxxxxxxxxxxxx.com/_assets/js/app.js?1650156247:478
    $emit https://wiki.xxxxxxxxxxxxxxxx.com/_assets/js/app.js?1650156247:478
    click https://wiki.xxxxxxxxxxxxxxxx.com/_assets/js/app.js?1650156247:478
    Wt https://wiki.xxxxxxxxxxxxxxxx.com/_assets/js/app.js?1650156247:478
    n https://wiki.xxxxxxxxxxxxxxxx.com/_assets/js/app.js?1650156247:478
    _wrapper https://wiki.xxxxxxxxxxxxxxxx.com/_assets/js/app.js?1650156247:478
    Xi https://wiki.xxxxxxxxxxxxxxxx.com/_assets/js/app.js?1650156247:478
app.js:478:11699
```

**Comment by Fringe-Admin** [comment]
I am having the same issue but I additionally cannot see previously posted comments.  On 2.5.278

**Comment by nicholas-alonzo** [comment]
Same problem on the latest version, past comments are not visible and you can't make a new comment

**Comment by hsoft-yulijia** [comment]
Same on 2.5.278. On 2.5.277 all works fine

**Comment by zdw1011781461** [comment]
#5200  Under advice to see

**Comment by NGPixel** [comment]
Fixed in 2.5.279

**Comment by zdw1011781461** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。
